tests/integration/test_vulnerability_detector/test_scan_types/test_partial_scan_type.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+import time
+
+import pytest
+
+import wazuh_testing.vulnerability_detector as vd
+from wazuh_testing.tools import ALERT_LOGS_PATH, LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+pytestmark = [pytest.mark.server, pytest.mark.tier(level=0)]
+
+local_internal_options = {
+    'wazuh_modules.debug': 2,
+    'monitord.rotate_log': 0
+}
+
+# variables
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_feed_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'data', 'feeds')
+configurations_path = os.path.join(test_data_path, 'wazuh_partial_scan_type.yaml')
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+wazuh_alert_monitor = FileMonitor(ALERT_LOGS_PATH)
+test_packet_vendor = 'WazuhIntegrationTests'
+test_packet_version = '1.0.0'
+test_packet_0_name = 'wazuhintegrationpackage-0'
+test_packet_1_name = 'wazuhintegrationpackage-1'
+test_packet_0_cve = 'CVE-000'
+test_packet_1_cve = 'CVE-001'
+
+# Offline feeds
+buster_oval_feed_path = os.path.join(test_feed_path, vd.CUSTOM_DEBIAN_OVAL_FEED)
+debian_json_feed_path = os.path.join(test_feed_path, vd.CUSTOM_DEBIAN_JSON_FEED)
+nvd_json_feed_path = os.path.join(test_feed_path, vd.CUSTOM_NVD_FEED)
+parameters = [{
+    'BUSTER_FEED_PATH': buster_oval_feed_path,
+    'DEBIAN_JSON_FEED_PATH': debian_json_feed_path,
+    'NVD_JSON_FEED_PATH': nvd_json_feed_path
+}]
+metadata = [{
+    'BUSTER_FEED_PATH': buster_oval_feed_path,
+    'DEBIAN_JSON_FEED_PATH': debian_json_feed_path,
+    'NVD_JSON_FEED_PATH': nvd_json_feed_path
+}]
```

Comment:
```suggestion
parameters = [{
    'BUSTER_FEED_PATH': buster_oval_feed_path,
    'DEBIAN_JSON_FEED_PATH': debian_json_feed_path,
    'NVD_JSON_FEED_PATH': nvd_json_feed_path
}]
metadata = parameters
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1428/commits/d0b7c1dfdf179be264e9fde70337ad9223ca8d3f